Werewolf for Telegram/Werewolf Control/Commands/AdminCommands.cs

```diff
@@ -23,6 +23,114 @@ namespace Werewolf_Control
+            }
+            else
+            {
+                infoText += $"> You are currently inside topic ID: `{topicId}`.\n" +
+                            "Choose what you'd like to do:";
+            }
+
+            var buttons = new List<InlineKeyboardButton[]>();
+
+            if (!inGeneralTopic)
+            {
+                buttons.Add(new[]{
+                    InlineKeyboardButton.WithCallbackData("Set Topic", "setgrouptopic_cmd|set"),
+                });
+            }
+            
+            buttons.Add(new[]{
+                InlineKeyboardButton.WithCallbackData("Unset Topic", "setgrouptopic_cmd|unset")
+            });
+            
+
+            var inlineKeyboard = new InlineKeyboardMarkup(buttons);
+
+            Bot.Send(
+                infoText,
+                chatId,
+                customMenu: inlineKeyboard,
+                parseMode: ParseMode.Markdown,
+                messageThreadId: topicId,
+                forceTopic: false // TODO: use this param in other admin cmd, cmd which admin are allowed to use anywhere
+            );
+        }
+
+        internal static void SetGroupTopicCallback(CallbackQuery query)
+        {
+            var chatId = query.Message.Chat.Id;
+            var topicId = query.Message.MessageThreadId;
+
+            using (var db = new WWContext())
+            {
+                var group = db.Groups.FirstOrDefault(g => g.GroupId == chatId);
+                if (group == null)
+                {
+                    group = MakeDefaultGroup(chatId, query.Message.Chat.Title, "setgrouptopic_callback");
+                    db.Groups.Add(group);
+                }
+
+                string[] args = query.Data.Split('|');
+                string action = args.Length > 1 ? args[1] : "";
+
+                switch (action)
+                {
+                    case "set":
+                        if (topicId == null)
+                        {
+                            Bot.ReplyToCallback(query, "This must be used inside a topic/thread.");
+                            return;
+                        }
+
+                        group.GroupTopicId = topicId;
+                        db.SaveChanges();
+
+                        Bot.Api.EditMessageTextAsync(chatId, query.Message.MessageId,
+                            "Group topic has been set successfully.");
+                        break;
+
+                    case "unset":
+                        group.GroupTopicId = null;
+                        db.SaveChanges();
+
+                        Bot.Api.EditMessageTextAsync(chatId, query.Message.MessageId,
+                            "Group topic has been unset.");
+                        break;
+
+                    default:
+                        Bot.ReplyToCallback(query, "Invalid topic action.");
+                        break;
+                }
+
+                Bot.Api.AnswerCallbackQueryAsync(query.Id);
+            }
+        }
+
+
```

Comment:
Instead of this callback button approach, I think `/settopic` and `/remtopic` commands would be preferred here, to keep the interface consistent with existing `/setlink` / `/remlink` commands. No need for callback buttons then.
Also please make sure to make the strings localized again.